src/jit/codegencommon.cpp

```diff
@@ -10241,6 +10241,62 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 // Look in CodeGenArm64.cpp
 
+#elif defined(_TARGET_X86_)
+
+/*****************************************************************************
+ *
+ *  Generates code for an EH funclet prolog.
+ */
+
```

Comment:
I currently have no idea on which action will be required for prolog/epilog. Could you let me know if you have any idea?